package.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
this probably changed how can a developer get started on this project? can you also update the readme doc?

Reply:
Done in [57fa2dd](https://github.com/mysterytony/timelocker-chrome/pull/9/commits/57fa2ddd3797aef5870cbd873b6e0f71f7e00f03)